README.md

```diff
@@ -29,3 +32,9 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons of pancakes? What is it in terms of the number of flips?
 Add your answer to this markdown file.
 
+The outer loop runs n times along with each of the three inner loops. This provides
+$T(3n^2) \in \Theta(n^2)$  as the runtime complexity based om the number of compares. The number of flips would be 2n
```

Comment:
So is this the runtime complexity or the complexity in terms of number of comparisons?